src/azul/health.py

```diff
@@ -202,3 +215,39 @@ def response(self):
     def generate_cache(self):
         health_object = dict(time=time.time(), health=self.health.as_json())
         self.storage_service.put(object_key=f'health/{self.health.lambda_name}', data=json.dumps(health_object))
+
+    def archive_fail_messages(self):
+        client = boto3.client('dynamodb')
+        fail_queue = boto3.resource('sqs').get_queue_by_name(QueueName=config.fail_queue_name)
+        while True:
+            received_messages = fail_queue.receive_messages(AttributeNames=['All'],
+                                                            MaxNumberOfMessages=10,
+                                                            WaitTimeSeconds=20,
+                                                            VisibilityTimeout=300)
+            if received_messages:
+                message_batch = [{
```

Comment:
Once we're in a wrapped literal or comprehension we should wrap it completely and not join brackets `[{`, here and everywhere else you joined brackets.